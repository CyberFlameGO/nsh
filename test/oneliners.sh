ls | echo hello | hexdump -C
for x in 1 2 "3 4"; do echo $x; done
{ echo hello; echo world; foo=123 env | grep foo; } | hexdump -C

msg1=guten
( echo $msg1; msg1=tag; echo $msg1 ) | hexdump -C
echo $msg1
