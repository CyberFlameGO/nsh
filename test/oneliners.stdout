00000000  68 65 6c 6c 6f 0a                                 |hello.|
00000006
1
2
3 4
00000000  68 65 6c 6c 6f 0a 77 6f  72 6c 64 0a 66 6f 6f 3d  |hello.world.foo=|
00000010  31 32 33 0a                                       |123.|
00000014
00000000  67 75 74 65 6e 0a 74 61  67 0a                    |guten.tag.|
0000000a
guten
